maquetacion/src/scss/_navbar.scss

```diff
@@ -0,0 +1,28 @@
+@import 'variables';
+
+.nav {
+  border-top: 6px solid $wolox-blue;
+  box-shadow: 0 4px 10px 0 $half-black;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+  
+  &__content {
+    align-items: center;
+    display: flex;
+    height: 70px;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__logo {
+    height: 70px;
+    width: 212.42px;
```

Comment:
We usually don't use decimals for sizes since it makes no difference. Just `212px` should be fine 😄

Reply:
Yep, I changed to 215px